assets/css/_announcements-interests-nav.scss

```diff
@@ -1,6 +1,10 @@
 .segmented-control {
+  background-color: $white;
   border-bottom: $base-border;
   font-size: $small-font-size;
+  line-height: $base-spacing;
+  position: relative;
+  z-index: $base-z-index + 1;
```

Comment:
Does it make sense to use a var for this instead? I just took a look at the z-indexes we have and seems like we need an overhaul/more variation, but that may be a PR for another time.

Reply:
Yup, I'll investigate and maybe refactor now.

Reply:
This may be a little heavy-handed for this project, but here's a map I often use for z-indexes (mostly to keep things agnostic and in context). Values Eric Bailey originally concocted actually!

```sass
$z-index: (
  basement: -999999,
  layout: 1000,
  object: 2000,
  component: 3000,
  subcomponent: 3010,
  modal: 4000,
  submodal: 4010,
  ceiling: 999999,
);
```

And then use this function:
```sass
@function get-z-index($index) {
  @return map-get($z-index, $index);
}
```

In use:
```sass
.foo-modal {
  z-index: get-z-index("modal");
}
```

Reply:
@enatario thoughts on this approach
https://github.com/thoughtbot/constable/pull/457/commits/3220c7c8a65175f974937ba2e8e18c6f30e8ed4e

Reply:
fab 🎉

Reply:
Oh I dig that map, though I agree it's overkill for this. I'm going to make a gist of that!